nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -64,6 +64,10 @@ def test_find_cut_slices():
                     cut_value = data[..., cut]
                 assert_equal(cut_value.max(), 1)
 
+    for n_cuts in (0, -2):
```

Comment:
I would create a separate test for this error message and use assert_raises_regex from somewhere in utils to check the content of the error message.